### PR TITLE
ref(analytics) Convert to TS

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by robertpin** [comment]
Jenkins test this please

File: react/features/analytics/handlers/AbstractHandler.ts [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+    _whiteListedEvents: Array<string>;
+    _blackListedEvents: Array<string>;
+
     /**
      * Creates new instance.
      *
      * @param {Object} options - Optional parameters.
      */
-    constructor(options = {}) {
+    constructor(options: any = {}) {
```
[the remark this answers is not in the document]
Comment (robertpin) [reply react/features/analytics/handlers/AbstractHandler.ts]: Done

**Review by gabiborlea — APPROVED** [review]

**Comment by nomadbyte** [comment]
Not sure how this commit passed the CI checks before the merge, but currently it's bombing the CI for all new PRs at this line:

https://github.com/jitsi/jitsi-meet/blob/a87d4f4a1dc9aa1816bfa9b37fa526f10eb304dd/react/features/analytics/AnalyticsEvents.ts#L588

Spaces are needed after `{` and before `}` in the inner arg type definition.

**Comment by saghul** [comment]
I think it passed because it was written before the other PR which added the rule.

@robertpin can you please take care of this?

**Pull request merged**